### DISCUSSION TITLE
PDFCreator

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#16] [Requests] [answered]
PDFCreator website: https://www.pdfforge.org/pdfcreator

PDFCreator privacy policy: https://www.pdfforge.org/about-us/privacy-policy

Logo: https://upload.wikimedia.org/wikipedia/en/8/84/PDF_Forge_PDFCreator_Free_Logo.png

Description: "PDFCreator is an application for converting documents into Portable Document Format (PDF) format on Microsoft Windows operating systems. It works by creating a virtual printer that prints to PDF files, and thereby allows practically any application to create PDF files by choosing to print from within the application and then printing to the PDFCreator printer.

Since 2009, PDFCreator has included closed source adware, toolbars and other controversial software that is installed by default."

**Comment by mayrinck** [comment, marked as answer]
> Since 2009, PDFCreator has included closed source adware, toolbars and other controversial software that is installed by default.

This might be the only reason I found not to post this one there... this thing of adwares and stuff...

**Reply by JuliaAgostino** [reply]
> > Since 2009, PDFCreator has included closed source adware, toolbars and other controversial software that is installed by default.
> 
> This might be the only reason I found not to post this one there... this thing of adwares and stuff...

Okay, that's a fair point.
